src/ITensorAutoHOOT/utils.jl

```diff
@@ -0,0 +1,210 @@
+using ITensors, AutoHOOT
+
+const ad = AutoHOOT.autodiff
+
+function get_symbol(i::Int)
+  if i < 26
+    return 'a' + i
+  elseif i < 52
+    return 'A' + i - 26
+  end
+  return Char(i + 140)
+end
+
+"""
+Retrieve the key from the dictionary that maps AutoHOOT nodes
+to ITensor tensors. Returns Nothing if key not exists.
+"""
+function retrieve_key(dict, value)
```

Comment:
Note that you can use `findfirst` for this:
```julia
julia> findfirst(==(2), Dict(:a => 1, :b => 2))
:b

julia> findfirst(==(3), Dict(:a => 1, :b => 2)) # returns nothing

```

Reply:
cool, will update

Reply:
Done